yt/utilities/lib/field_interpolation_tables.pxd

```diff
@@ -10,7 +13,8 @@ cimport cython
 cimport numpy as np
 from yt.utilities.lib.fp_utils cimport imax, fmax, imin, fmin, iclip, fclip, fabs
 from libc.stdlib cimport malloc
-from libc.math cimport isnormal
+cdef extern from "<cmath>" namespace "std":
+    bint isnormal(double x) nogil
```

Comment:
Note: for some reason, Visual Studio doesn't happy about this. However, if we replace by
```suggestion
cdef extern from "<math.h>" namespace "std":
    bint isnormal(double x) nogil
```
it compiles.